samples/event-api-handlers/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM amazoncorretto:17-alpine3.18@sha256:6a3dc573328ced158ae25ffde792accbe8b1ff2e7b505c72d2d5e7b9bfa9ac26
+FROM amazoncorretto:21-alpine3.18@sha256:6a3dc573328ced158ae25ffde792accbe8b1ff2e7b505c72d2d5e7b9bfa9ac26
```

Comment:
Please verify the image SHA

Reply:
Confirmed with devops to use the same SHA image